pulsar-common/src/main/java/org/apache/pulsar/common/stats/JvmDefaultGCMetricsLogger.java

```diff
@@ -36,19 +40,46 @@
     private volatile long accumulatedFullGcTime = 0;
     private volatile long currentFullGcTime = 0;
 
-    @SuppressWarnings("restriction")
-    private static sun.management.HotspotRuntimeMBean runtime;
+    private static Object /*sun.management.HotspotRuntimeMBean*/ runtime;
+    private static Method getTotalSafepointTimeHandle;
```

Comment:
nit: naming this `getTotalSafepointTimeMethod` instead of `*Handle`? Since there's also [java.lang.invoke.MethodHandle](https://docs.oracle.com/javase/8/docs/api/java/lang/invoke/MethodHandle.html) which is different than `java.lang.reflect.Method`.

Reply:
I tried with method hanldes but I was not able to access the method.
So I did fall back to old reflection style